tests/kubeUtils.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Copyright 2017 Istio Authors
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+# Bring up control plane
+deploy_istio() {
+    print_block_echo "Deploying ISTIO"
+    $K8CLI create -f istio/controlplane.yaml
+    for (( i=0; i<=4; i++ ))
+    do
+        ready=$($K8CLI get pods | awk 'NR>1 {print $1 "\t" $2}' | grep "istio" | grep "1/1" | wc -l)
+        if [ $ready -eq 2 ]
+        then
+            echo "ISTIO control plane deployed"
+            return 0
+        fi
+        sleep 10
+    done
+    echo "Unable to deploy ISTIO"
+    return 1
+}
+
+# Clean up the controlplane when we are done with it
+cleanup_istio(){
```

Comment:
If you create a namespace and put everything in it, you can just delete the namespace I believe.